lbuild/config.py

```diff
@@ -220,9 +220,10 @@ def from_file(configfile, parent=None):
         config._modules = xmltree.xpath('modules/module/text()')
 
         # Load output path for lbuild and modules
-        outPaths = xmltree.xpath('outpath/text()')
-        config._outpath = outPaths[0] if outPaths else None 
-        
+        out_path = xmltree.find("outpath")
+        if out_path is not None:
+            config._outpath = ConfigNode._rel_path(out_path.text, configpath)
```

Comment:
Looks way cleaner, I was unsure about the XML APIso your code looks way better :)

I am unsure about the relative path thing though, because it is relative by default (to lbuild invocation) .. I thought that the config file (aka project.xml) has nothing to do with it, because lbuild can be invoked wherever